src/python-frontend/python_converter.cpp

```diff
@@ -545,7 +545,14 @@ function_id python_converter::build_function_id(const nlohmann::json &element)
   {
     is_member_function_call = true;
     func_name = func_json["attr"];
-    obj_name = func_json["value"]["id"];
+    if (func_json["value"]["_type"] == "Attribute")
+    {
```

Comment:
We try to avoid using curly braces for single-line if statements. &#128578;

Reply:
Done